webapp-django/crashstats/api/tests/test_views.py

```diff
@@ -183,27 +183,24 @@ def mocked_get(*args, **k):
         assert dump['hits'][0] == expected
 
     def test_metrics_gathering(self):
-        # note: this gets mocked out in the setUp
-        url = reverse('api:model_wrapper', args=('Platforms',))
+        def mocked_get(**options):
+            return {
+                'state': {}
+            }
+
+        CrontabberState.implementation().get.side_effect = mocked_get
+
+        url = reverse('api:model_wrapper', args=('CrontabberState',))
         with MetricsMock() as metrics_mock:
             response = self.client.get(url)
         assert response.status_code == 200
         assert metrics_mock.has_record(
             fun_name='incr',
             stat='webapp.api.pageview',
             value=1,
-            tags=['endpoint:apiPlatforms']
+            tags=['endpoint:apiCrontabberState']
```

Comment:
We're getting rid of the `/api/Platforms`, so I had to switch this to use `/api/CrontabberState`.